update-header/action.yaml

```diff
@@ -0,0 +1,70 @@
+name: "Update File headers to the new year"
+author: "Jaspar Löchte <jaspar.loechte@greenbone.net>"
+description: "An action that updates header year in all files of given directories."
+
+inputs:
+  github-user:
+    description: "Github user name."
+    required: true
+  github-user-mail:
+    description: "Mail address for the given github user."
+    required: true
+  github-user-token:
+    description: "Token with write rights required to create the release."
+    required: true
+  version:
+    description: "Python version that should be installed"
+    default: 3.9
+  target:
+    description: "Branch that should be released. Default: main"
+    default: 'main'
```

Comment:
add `required: false`

Reply:
Isn't `false` default here?

Reply:
https://json.schemastore.org/github-action.json say: it is required field

Reply:
```
...
  "inputs": {
      "$comment": "https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs",
      "description": "Input parameters allow you to specify data that the action expects to use during runtime. GitHub stores input parameters as environment variables. Input ids with uppercase letters are converted to lowercase during runtime. We recommended using lowercase input ids.",
      "type": "object",
      "patternProperties": {
        "^[_a-zA-Z][a-zA-Z0-9_-]*$": {
          "$comment": "https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_id",
          "description": "A string identifier to associate with the input. The value of `<input_id>` is a map of the input's metadata. The `<input_id>` must be a unique identifier within the inputs object. The `<input_id>` must start with a letter or `_` and contain only alphanumeric characters, `-`, or `_`.",
          "type": "object",
          "properties": {
...
          "required": [
            "description",
            "required"
          ],
....